src/components/CityItem.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
- [ ] Also as mentioned previously, kindly consider looking for a way to correct your linters errors instead of disabling them for they help standardize your codebase and make it cleaner.